packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue

```diff
@@ -29,13 +32,26 @@ const emit = defineEmits<{
    * Emitted when the current search input value changes.
    */
   "update:searchTerm": [searchTerm: string];
+  "update:open": [open: boolean];
   /**
    * Emitted if lazy loading is triggered / the users scrolls to the end of the options.
    * See property `lazyLoading` for enabling the lazy loading.
    */
   lazyLoad: [];
 }>();
 
+const searchTerm = useManagedState(
+  toRef(() => props.searchTerm),
+  "",
+  (v) => emit("update:searchTerm", v ?? ""),
```

Comment:
The default value seems a little redudant here, maybe we can to this in the `useManagedState` internally?

Because it is passed as second parameter here and also for the `?? ""`

Reply:
The fallback value here is redundant, I removed it.